lineapy/graph_reader/program_slice.py

```diff
@@ -206,8 +210,10 @@ def get_source_code_from_graph(program: Graph) -> CodeSlice:
     source_code_to_lines = DefaultDict[SourceCode, Set[int]](set)
     import_code_to_lines = DefaultDict[SourceCode, Set[int]](set)
     incomplete_block_locations = DefaultDict[SourceCode, Set[int]](set)
+    session_src = DefaultDict[SourceCode, Set[LineaID]](set)
```

Comment:
Logic is pretty clean, but maybe a quick comment indicating that session_src _does not_ always contain all the session code, (only when include_non_slice_as_comment) due to L283.